### DISCUSSION TITLE
Nightly builds?

### DISCUSSION THREAD

**Discussion by hillin** [spartan-ng/spartan#502] [General]
Is there an easy way to retrieve the latest build? The packages on npm are already 2 months old.

**Comment by elite-benni** [comment]
We are in the preparation phase for regularly automatic builds. 
There are some changes coming regarding the project structure, in the future we will only have one brn package. 

This week a new version was published.

**Reply by maxhov** [reply]
hi @elite-benni are there any new insights on this? When do you think we could access those nightly builds or what is the generic plan on releasing?

**Reply by elite-benni** [reply]
Hi @maxhov the nightly builds are already live. Releases automatically happen if the github action sees a commit. 

We are pretty close to release 1.0, but there is no planned release date yet. 
For V1 we want to move to angular 19 and merge the remaining pull requests.
We are also working on a update routine for helm and improved migrations. 
Addtionally we are discussing internally which of the https://github.com/orgs/spartan-ng/projects/1/views/1 are really mandatory for V1.

**Reply by maxhov** [reply]
thanks for your fast reply @elite-benni. I noticed the CI indeed, but I cannot figure out where the nightly's are pushed. Could you explain where from I could get those?

**Reply by goetzrobin** [reply]
They are releasing a new alpha to NPM if there was changes. Until we go live with V1, then we will probably set up a dedicated nightly tag. Does that make sense?

**Reply by maxhov** [reply]
Not sure I understand what you mean. In npm the latest alpha release is 374, released on 9th of December. If there's anything I can contribute here I am happy to do so.

**Reply by ashley-hunter** [reply]
What package are you looking at? If we take the brain package, for example, it was published 13 hours ago.

https://www.npmjs.com/package/@spartan-ng/brain

**Reply by maxhov** [reply]
I was hoping to get this change: https://github.com/spartan-ng/spartan/commit/0395d751ac319f26d69e23a7830216fc20496fac , which I think should (at least) release this package: https://www.npmjs.com/package/@spartan-ng/ui-dialog-brain but it was last updated 3 months ago. At least, I only have that package installed when I install the dialog as described here: https://www.spartan.ng/components/dialog

**Reply by ashley-hunter** [reply]
Ah, we consolidated all the brain packages in to a single package, so that package you are looking at is no longer updated. 

If you update the CLI package to the latest version and run out health check command it will migrate everything for you automatically:

https://www.spartan.ng/documentation/health-checks

**Reply by maxhov** [reply]
That did it! Thanks for that. Now just have to migrate the changes.

**Reply by elite-benni** [reply]
Perfect, Thanks @ashley-hunter and @goetzrobin for jumping in. ;)

**Reply by ashley-hunter** [reply]
No problem 🙂 now hopefully the migrations work well! Any issues let us know! Thanks
